src/inmanta/env.py

```diff
@@ -767,6 +838,43 @@ def install_for_config(
             paths=paths,
         )
 
+    async def async_install_for_config(
+        self,
+        requirements: list[Requirement],
+        config: PipConfig,
+        upgrade: bool = False,
+        constraint_files: Optional[list[str]] = None,
+        upgrade_strategy: PipUpgradeStrategy = PipUpgradeStrategy.ONLY_IF_NEEDED,
+        paths: list[LocalPackagePath] = [],
+    ) -> None:
+        """
+        Perform a pip install in this environment, according to the given config
+
+        :param requirements: which requirements to install
+        :param paths: which paths to install
+        :param config: the pip config to use
+        :param constraint_files: pass along the following constraint files
+        :param upgrade: make pip do an upgrade
+        :param upgrade_strategy: what upgrade strategy to use
```

Comment:
The order of attributes in the docstring is inconsistent with the order in the method declaration.
```suggestion
        :param requirements: which requirements to install
        :param config: the pip config to use
        :param upgrade: make pip do an upgrade
        :param constraint_files: pass along the following constraint files
        :param upgrade_strategy: what upgrade strategy to use
        :param paths: which paths to install
```